core/src/test/java/com/gentics/mesh/core/schema/field/BooleanListFieldMigrationTest.java

```diff
@@ -36,15 +32,6 @@ public void testRemove() throws Exception {
 		removeField(CREATEBOOLEANLIST, FILL, FETCH);
 	}
 
-	@Override
-	@Test
-	public void testRename() throws Exception {
```

Comment:
Check why this was removed.

Reply:
A custom script was used for renaming fields.